sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultSettingsClient.cs

```diff
@@ -196,7 +199,13 @@ public virtual async Task<Response<KeyVaultSetting>> UpdateSettingAsync(KeyVault
             scope.Start();
             try
             {
-                return await _restClient.UpdateSettingAsync(VaultUri.AbsoluteUri, setting.Name, setting.Value.ToString(), cancellationToken).ConfigureAwait(false);
+                Response response = await _restClient.UpdateSettingAsync(setting.Name, setting.Value.ToString()).ConfigureAwait(false);
+
+                KeyVaultSetting value0 = default;
```

Comment:
```suggestion
                KeyVaultSetting firstSetting = default;
```

nit: The naming on this is awkward.  We should make that more readable.